search/includes/classes/class-queue.php

```diff
@@ -507,7 +507,7 @@ public function empty_queue() {
 
 		$table_name = $this->schema->get_table_name();
 
-		return $wpdb->query( "TRUNCATE TABLE {$table_name}" ); // Cannot prepare table name. @codingStandardsIgnoreLine
+		return $wpdb->query( "DELETE FROM {$table_name}" ); // Cannot prepare table name. @codingStandardsIgnoreLine
```

Comment:
`TRUNCATE` breaks transactions (WPTL runs tests inside a transaction); this breaks some tests (they time out after 10 minutes of inactivity)